### PR TITLE
[3.3.3] Add operation IN and NOT_IN for String value type to key filter

### PR DESCRIPTION
**Pull request by van-vanich**
Now, for checking present or don't present element is a set of elements we need to use a complex filter. But, if we have many elements is too long. I think, the issue is to add operation IN and NOT_IN.
Operation IN and NOT_IN get a set of elements separated by a comma. Also, the user can take items to " " or ' '.
Example image data :
1. device1, device2
2. 'device1', 'device2'
3. "device1", "device2"
Old realization:
image
Now:
image

### CODE REVIEW

File: dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java [hunk 1]
```diff
@@ -560,8 +561,24 @@ private String buildStringPredicateQuery(QueryContext ctx, String field, StringF
...
                 break;
+            case IN:
+                value = value.replaceAll("'","").replaceAll("\"", "");
+                values = List.of(value.trim().split("\\s*,\\s*"));
+                stringOperationQuery = String.format("%s in (:%s))", operationField, paramName);
+                break;
+            case NOT_IN:
+                value = value.replaceAll("'","").replaceAll("\"", "");
+                values = List.of(value.trim().split("\\s*,\\s*"));
+                stringOperationQuery = String.format("%s not in (:%s))", operationField, paramName);
```
Comment (volodymyr-babak) [inline dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java:572]: break is missing

File: dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java [hunk 2]
```diff
@@ -537,6 +537,7 @@ private String buildStringPredicateQuery(QueryContext ctx, String field, StringF
             value = value.toLowerCase();
             operationField = String.format("lower(%s)", operationField);
         }
+        List<String> values = new ArrayList<>();
```
Comment (volodymyr-babak) [inline dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java:540]: let's remove this array that is not related to all cases

Comment (volodymyr-babak) [reply dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java:540]: we can create this array in second switch statement

Comment (volodymyr-babak) [reply dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java:540]: case IN:
            case NOT_IN:
                ctx.addStringListParameter(paramName, List.of(value.trim().split("\\s*,\\s*")));
                break;

File: dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java [hunk 3]
```diff
@@ -560,8 +560,23 @@ private String buildStringPredicateQuery(QueryContext ctx, String field, StringF
                 value = "%" + value + "%";
                 stringOperationQuery = String.format("%s not like :%s or %s is null)", operationField, paramName, operationField);
                 break;
+            case IN:
+                value = value.replaceAll("'","").replaceAll("\"", "");
```
Comment (ashvayka) [inline dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java:564]: Move 
```java
value.replaceAll("'","").replaceAll("\"", "")
```
to a method with a meaningful name. Also add 
```java
List.of(value.trim().split("\\s*,\\s*"))
```
there to avoid extra switch on line 572
